gelf/writer.go

```diff
@@ -46,16 +46,16 @@ const (
 // Message represents the contents of the GELF message.  It is gzipped
 // before sending.
 type Message struct {
-	Version    string                 `json:"version"`
-	Host       string                 `json:"host"`
-	Short      string                 `json:"short_message"`
-	Full       string                 `json:"full_message"`
-	TimeUnixMs int64                  `json:"timestamp_ms"`
-	Level      int32                  `json:"level"`
-	Facility   string                 `json:"facility"`
-	File       string                 `json:"file"`
-	Line       int                    `json:"line"`
-	Extra      map[string]interface{} `json:"-"`
+	Version  string                 `json:"version"`
+	Host     string                 `json:"host"`
+	Short    string                 `json:"short_message"`
+	Full     string                 `json:"full_message"`
+	TimeUnix float64                `json:"timestamp"`
```

Comment:
when we implemented the `timestamp_ms` it was working, not sure if graylog changed or the gelf parser it is using changed, either way, we should use this field now.